community/blog/2021-05 Enhanced request tracking in Arcus.WebApi v1.3/README.md

```diff
@@ -0,0 +1,135 @@
+# Enhanced request tracking in Arcus.WebApi v1.3
+
+The new version v1.3 of the [Arcus Web API](https://github.com/arcus-azure/arcus.webapi) project is all about request tracking. Previous versions introduced the request tracking concept as a way to track the incoming HTTP request and outgoing HTTP response with additional configurations on what should be included in the tracking information. This includes the request/response body, headers, the time it took to process the request...
+
+This new version takes this to a whole new level so that your next project can benefit from a production-ready request tracking functionality!
+
+## Omit entire routes from the request tracking
+
+Initially, the request tracking was tracking all routes. This was a good first basic approach but when more realistic scenario's comes into play, this is rather cumbersome.
+For example: health endpoints to track the health of your web API could add noise to the tracking. Especially when their called often, and the result is almost always success.
+
+We created an `[ExcludeRequestTracking]` attribute that you can place upon routes (both on class and method level). This attribute will exclude the entire request tracking functionality from that specific route:
+
+```csharp
+using Arcus.WebApi.Logging;
+using Microsoft.AspNetCore.Mvc;
+
+[ApiController]
+[Route("/api/v1/health")]
+public class HealthController : ControllerBase
+{
+    [HttpGet]
+    [ExcludeRequestTracking]
+    public IActionResult Get()
+    {
+        // Determine health status...
+        return Ok();
+    }
+}
+```
+
+When such a health endpoint is generated (like [Dapr](https://docs.dapr.io/developing-applications/building-blocks/observability/sidecar-health/)) and you have no control over where the this endpoint is hosted, you can't really add this attribute. For those scenarios we added an additional option to the configurable request tracking options:
+
+```csharp
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+
+public class Startup
+{
+    public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+    {
+        app.UseRequestTracking(options => options.OmitRoutes.Add("/api/v1/health"));
+    }
+}
+```
+
+## Only track certain HTTP status codes
+
+As a continuation of the previous health endpoints example. It would be even better if we only could see health failures and disregard all success statuses. That would be both make the tracking more cleaner and help with defect detection when something's wrong.
```

Comment:
```suggestion
As a continuation of the previous health endpoints example. It would be even better if we can only measure health failures and ignore all success statuses. That would be both make the tracking more cost-efficient, actionable and help with troubleshooting.
```